### PR TITLE
Added Car model

### PR DESCRIPTION
**Pull request by ArtemReizer**
added Car model and liquibase

### CODE REVIEW

File: src/main/java/com/jvavateam/carsharingapp/model/Car.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    private Long id;
+    @Column(nullable = false)
+    private String model;
+    @Column(nullable = false)
+    private String brand;
+    @Column(nullable = false)
+    private int inventory;
+    @Column(nullable = false)
+    private BigDecimal dailyFee;
+    @Column(nullable = false)
```
Comment (Dimagaa) [inline src/main/java/com/jvavateam/carsharingapp/model/Car.java:31]: @Enumerated(EnumType.STRING)

Comment (Dimagaa) [inline src/main/java/com/jvavateam/carsharingapp/model/Car.java:33]: @Enumerated(EnumType.STRING)

File: src/main/resources/db/changelog/changes/02.10.CreateCarsTable.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                  constraints:
+                    nullable: false
+              - column:
+                  name: inventory
+                  type: int
+                  constraints:
+                    nullable: false
+              - column:
+                  name: daily_fee
+                  type: numeric
```
Comment (Dimagaa) [inline src/main/resources/db/changelog/changes/02.10.CreateCarsTable.yaml]:
```suggestion
                  type: decimal(10, 2)
```

**Review by Dimagaa — CHANGES_REQUESTED** [review]
Good Job!  Just a few fixes

**Review by CodeWebWeaver — DISMISSED** [review]

**Review by CodeWebWeaver — APPROVED** [review]

**Review by Dimagaa — APPROVED** [review]

**Pull request merged**